tests/phpunit/tests/multisite/site.php

```diff
@@ -1674,6 +1680,18 @@ public function data_wp_normalize_site_data() {
 					),
 					array(),
 				),
+				array(
+					array(
+						'domain' => 'domainwithport.com:443',
+						'domain' => 'anotherdomainwithport.com:80',
+						'domain' => 'anotherwithport.com:8080',
+					),
+					array(
+						'domain' => 'domainwithport.com:443',
+						'domain' => 'anotherdomainwithport.com:80',
+						'domain' => 'anotherwithport.com:8080',
+					),
+				),
```

Comment:
The `'domain'` key will be overwritten with the last value in each of these arrays.

These should be separated, see the `<yet>/another-invalid-domain.com` test above for an example.

```suggestion
				array(
					array(
						'domain' => 'domainwithport.com:443',
					),
					array(
						'domain' => 'domainwithport.com:443',
					),
				),						
				array(
					array(
						'domain' => 'anotherdomainwithport.com:80',
					),
					array(
						'domain' => 'anotherdomainwithport.com:80',
					),
				),	  
				array(
					array(
						'domain' => 'anotherwithport.com:8080',
					),
					array(
						'domain' => 'anotherwithport.com:8080',
					),
				),
```